### PR TITLE
Example works with plain cloudformation instead of cfndsl

### PR DESCRIPTION
**Pull request by pburkholder**
I've updated the examples to work with current versions in inspec and this project.

`cfndsl` was producing bogus json. with AWS::APIGateway resources instead of SecurityGroups, so I've gone to straight up CF json.

Update the profile and example to work, as show.

HTH, Peter

### CODE REVIEW

File: examples/profile/controls/test.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@
   impact 1.0
   title 'Security Group: No ingress access to CIDR block 0.0.0.0/0'
   desc 'Security Groups must not allow inbound access from anywhere'
-  
+
   Vpc.new(id: ENV['vpc_id']).security_groups.each do |security_group|
-    describe security_group do
```
Comment (arothian) [inline examples/profile/controls/test.rb:8]: This is designed to be able to work as previously written. Where you having issues with this line specifically?

**Comment by arothian** [comment]
@pburkholder Thanks for the PR and updated documentation. Sorry about the delay in reviewing.

I'm not seeing the same issue with cfndsl. The output of `bundle exec cfndsl examples/noncompliant_vpc_cfndsl.rb` includes APIGateway resources?

I commented inline, but I think the each on the VPC resource should already be creating the inspec resources for checking.

**Comment by pburkholder** [comment]
@arothian 

Indeed, when I blew away my vendor/bundle and reinstalled then it was ok, at least in terms of getting the correct json generated from the cfndsl input, and creating the stack in AWS.

The .gemspec seems broken for someone trying to get a quick test run, can you fix that?

This fails:

```
$  bundle exec examples/bin/demo_noncompliant_vpc.rb
Launching CFN stack InspecAwsDemo-NonCompliant-20160725191521 in us-west-2
Waiting for stack to come up...
Stack created - vpc-id[vpc-49703a2d]
Running Inspec on example profile...
Deleting stack InspecAwsDemo-NonCompliant-20160725191521
bundler: failed to load command: examples/bin/demo_noncompliant_vpc.rb (examples/bin/demo_noncompliant_vpc.rb)
NoMethodError: undefined method `add_tests' for #<Inspec::Runner:0x007fd37c5a8848>
  examples/bin/demo_noncompliant_vpc.rb:50:in `<top (required)>'
```

because, I think, some of the methods in Inspec have changed since you first wrote this.

That said, I think @chris-rock and @arlimus are going to have a plugin api soon so the approach to testing AWS will change, so I'm not investing more work in this myself.

**Comment by chris-redekop** [comment]
I looked for a good set of AWS resources for InSpec and found your repository, @arothian .  Is it still active, or has AWS testing with InSpec evolved to a new plugin API by @chris-rock and @arlimus, as suggested by @pburkholder ?

Does anyone have any news to share on AWS testing with InSpec?  I didn't find a better or more up-to-date approach than the one here...

**Comment by arothian** [comment]
Hey @chris-redekop , I'm not aware of another source for AWS inspec resources. This is a bit out of date. It really needs to be reworked with Inspec 1.0 release. They added better support for pulling in external resources more effectively. It will still need you to manually add in the AWS sdk dependencies.

I'm still very much interesting in seeing this happen, but I haven't had time yet to take a look at migrating what I have here to work with the new version of Inspec. I'd be happy to look at any PRs if you want to give it a try, or open an issue for the migration and we can discuss more. Maybe I'll find some time this week to get started on refactoring it.

**Comment by chris-redekop** [comment]
Thanks for the response, @arothian .  I'll follow up in the existing issue, and also create a new one.

**Comment by aaronlippold** [comment]
I am hoping that we will keep developing this resource. I will see if @chris-rock and @arlimus agree's. I think developing a profile that tests aws setups - i.e. the new AWS NIST layout - would be a great example. https://aws.amazon.com/quickstart/architecture/accelerator-nist/

**Comment by aaronlippold** [comment]
Right now AWS doesn't have a way to 'show' or 'prove' everything is as they said it is. I'd love a way to find a way to show it independably.

**Pull request closed without merging**